### ISSUE TITLE
Create endpoint to get the create content url

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#19683]
To create a contentlet, we get the url from the JSP, in this function: https://github.com/dotCMS/core/blob/master/dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets_js_inc.jsp#L825

We need to create an endpoint we pass the "Content Type Variable" and we get this link..

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19690

**Comment by alfredo-dotcms** [comment]
*Note: in site browser we need to add support to the endpoint to receive a folder param.

url built: 
```
var loc='<portlet:actionURL windowState="<%= WindowState.MAXIMIZED.toString() %>"><portlet:param name="struts_action" value="/ext/contentlet/edit_contentlet" /><portlet:param name="cmd" value="new" /></portlet:actionURL>&selectedStructure=' + selected +'&folder='+folderInode+'&referer=' + escape(refererVar);
```

**Comment by alfredo-dotcms** [comment]
*Note: In calendar we also need date1 & date2.
```
var addURL = '<portlet:actionURL windowState="<%= WindowState.MAXIMIZED.toString() %>">
					<portlet:param name="cmd" value="new" />
					<portlet:param name="struts_action" value="/ext/calendar/edit_event" />
					<portlet:param name="selectedStructure" value="<%= eventStructure.getInode() %>" />
					<portlet:param name="referer" value="<%= referer %>" />
					<portlet:param name="inode" value="" />
				</portlet:actionURL>&date1=' + startDate + '&date2=' + endDate;
```

**Comment by alfredo-dotcms** [comment]
*Note: in SITES, we need a bunch of params.
https://github.com/dotCMS/core/blob/b5d7b3c52420824b518c9e7445453dcfdde397e6/dotCMS%2Fsrc%2Fmain%2Fwebapp%2Fhtml%2Fportlet%2Fext%2Fhostadmin%2Fview_hosts_js_inc.jsp#L310

**Comment by erickgonzalez** [comment]
This will be done when migrating each portlet to angular, right now this endpoint works for the purpose that was meant (getting started portlet).

**Comment by bryanboza** [comment]
We need more postman test here: 
Example: 
- Validate response with wrong credentials 
- Validate sending an non existing content type 
- Validate if it is case sensitive

**Comment by erickgonzalez** [comment]
PR for tests requested: https://github.com/dotCMS/core/pull/20077

**Comment by bryanboza** [comment]
Fixed, tested on release-21.03 // Now we have more postman tests
